demisto_sdk/tests/workflow_test.py

```diff
@@ -340,12 +347,12 @@ def test_workflow_by_sequence(function: Callable):
         Create/Modify files:
             The test will create new files or will modify them.
         Run Demisto-SDK commands:
-            Will run any tested demisto-sdk functionality. as the init command and update-release-notes.
-        Run validations:
+            Will run any tested demisto-sdk functionality. as the init com
             Will run all validation with expected the test to pass.
             * secrets
             * lint -g --no-test
             * validate -g
     """
     global content_git_repo
-    function(content_git_repo)
+    monkeypatch.chdir(content_git_repo.content)  # type: ignore
```

Comment:
please add a dev comment on what is this monkeypatch as it is unique to this test.

Reply:
@jochman

Reply:
did in the docstring